doc/environment_variables.md

```diff
@@ -118,10 +118,10 @@ some configuration as environment variables that can be set.
     perform name resolution
   - ares - a DNS resolver based around the c-ares library
 
-* GRPC_DISABLE_CHANNEL_CONNECTIVITY_WATCHER
-  The channel connectivity watcher uses one extra thread to check the channel
-  state every 500 ms on the client side. It can help reconnect disconnected
-  client channels (mostly due to idleness), so that the next RPC on this channel
-  won't fail. Set to 1 to turn off this watcher and save a thread. Please note
-  this is a temporary work-around, it will be removed in the future once we have
-  support for automatically reestablishing failed connections.
+* GRPC_CLIENT_CHANNEL_BACKUP_POLL_INTERVAL_MS
+  Default: 500
```

Comment:
What is the justification for a default of 500 ms ? I would expect a value of 5seconds to be good enough. 

This feature is for correctness and I would like it to trigger as little as possible while maintaining the correctness objectives.

Reply:
Changed to 5s. I agree that 5s would be a safer number to use without hurting correctness. But if the concern is about the performance impact, our implementation here has minimized its performance impact. I've tested it with our benchmark suite. It had similar scores with 500ms, 5s, and Inf intervals. The differences between their scores are less than 0.005.